simulation/traffic_simulator/include/traffic_simulator/traffic_lights/traffic_light_manager_base.hpp

```diff
@@ -81,61 +91,33 @@ class TrafficLightManagerBase
 
   auto getTrafficLights() -> auto & { return traffic_lights_; }
 
-  auto getTrafficRelationReferees(const LaneletID lanelet_id)
+  auto getTrafficLights(const LaneletID lanelet_id)
     -> std::vector<std::reference_wrapper<TrafficLight>>
   {
-    std::vector<std::reference_wrapper<TrafficLight>> refers;
+    std::vector<std::reference_wrapper<TrafficLight>> traffic_lights;
 
     if (hdmap_->isTrafficRelation(lanelet_id)) {
       for (auto && traffic_light : hdmap_->getTrafficRelation(lanelet_id)->trafficLights()) {
-        refers.emplace_back(getTrafficLight(traffic_light.id()));
+        traffic_lights.emplace_back(getTrafficLight(traffic_light.id()));
       }
     } else if (hdmap_->isTrafficLight(lanelet_id)) {
-      refers.emplace_back(getTrafficLight(lanelet_id));
+      traffic_lights.emplace_back(getTrafficLight(lanelet_id));
     } else {
       throw common::scenario_simulator_exception::Error(
         "Given lanelet ID ", lanelet_id,
         " is neither a traffic light ID not a traffic relation ID.");
     }
 
-    return refers;
+    return traffic_lights;
   }
 
   auto hasAnyLightChanged() -> bool;
 
-  auto update(const double) -> void;
-};
+  auto start(double update_rate) -> void;
 
-template <typename Message>
-class TrafficLightManager : public TrafficLightManagerBase
-{
-  const typename rclcpp::Publisher<Message>::SharedPtr traffic_light_state_array_publisher_;
+  auto updatePublishRate(double update_rate) -> void;
```

Comment:
In the existing code of traffic_simulator, "update" is a verb assigned exclusively to a subroutine called every frame. So avoid giving the word "update" to functions that behave differently.

Reply:
use `apply` instead of `update` in https://github.com/tier4/scenario_simulator_v2/pull/1004/commits/9b7a262597ea86ff02ce771a0db7095d19dd3cfa

Reply:
use `reset` instead of `update` in https://github.com/tier4/scenario_simulator_v2/pull/1004/commits/76374fcd98686c413b3e90e8851ddfa9fb78b5d2